vulnerabilities/pipelines/v2_improvers/compute_advisory_todo.py

```diff
@@ -0,0 +1,347 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# VulnerableCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/aboutcode-org/vulnerablecode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+import json
+
+from aboutcode.pipeline import LoopProgress
+from django.utils import timezone
+
+from vulnerabilities.models import AdvisoryAlias
+from vulnerabilities.models import AdvisoryToDoV2
+from vulnerabilities.models import AdvisoryV2
+from vulnerabilities.models import ToDoRelatedAdvisoryV2
+from vulnerabilities.pipelines import VulnerableCodePipeline
+from vulnerabilities.pipes.advisory import advisories_checksum_v2
+
+
+class ComputeToDo(VulnerableCodePipeline):
+    """Compute ToDos for Advisory."""
+
+    pipeline_id = "compute_advisory_todo_v2"
+
+    @classmethod
+    def steps(cls):
+        return (
+            cls.compute_individual_advisory_todo,
+            cls.detect_conflicting_advisories,
+        )
+
+    def compute_individual_advisory_todo(self):
+        """Create ToDos for missing summary, affected and fixed packages."""
+
+        advisories = AdvisoryV2.objects.all()
```

Comment:
Since we're using impacted package in `check_missing_affected_and_fixed_by_packages` we should prefetch those here itself.